pkg/meta/dump.go

```diff
@@ -587,6 +596,11 @@ func decodeEntry(dec *json.Decoder, parent Ino, cs *DumpedCounters, parents map[
 			return nil, fmt.Errorf("decode %v: %s", name, err)
 		}
 	}
+	if decodeCounter >= gcThreshold || len(e.CompressedEntries) > gcThreshold {
+		decodeCounter = 0
+		runtime.GC()
```

Comment:
it's better to use mem pool

Reply:
yes, mem pool can improve performance and reduce GC pressure. 
However, testing shows that in this scenario, with 11M files in a single folder, if GC is not triggered in time, memory will still continue to grow, leading to OOM.

Reply:
We don't need to over optimize for this single case.

btw, since we don't use JSON to serialize the whole tree directly, we can change type of Entries of DumpedEntry to slice of compressAttr.

Reply:
btw, we can optmize this after 1.2

Reply:
> We don't need to over optimize for this single case.
> 
> btw, since we don't use JSON to serialize the whole tree directly, we can change type of Entries of DumpedEntry to slice of compressAttr.

`Entries           map[string]*DumpedEntry `json:"entries,omitempty"``
This variable is still necessary during the dump and load processes.